apps/wpcom-block-editor/src/wpcom/features/add-tags-education-link.js

```diff
@@ -0,0 +1,33 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ExternalLink } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+import { useTranslate } from 'i18n-calypso';
+import tracksRecordEvent from './tracking/track-record-event';
+
+const trackTagsEducationLinkClick = () =>
+	tracksRecordEvent( 'wpcom_block_editor_tags_education_link_click' );
+
+const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) => {
+	return ( props ) => {
+		const translate = useTranslate();
+
+		if ( props.slug !== 'post_tag' ) {
+			return <PostTaxonomyType { ...props } />;
+		}
+
+		return (
+			<>
+				<PostTaxonomyType { ...props } />
+				<ExternalLink
+					href={ localizeUrl( 'https://wordpress.com/support/posts/tags/' ) }
+					onClick={ trackTagsEducationLinkClick }
+				>
+					{ translate( 'Build your audience with tags' ) }
```

Comment:
When we're in the GB we need to use some different translation functions. They fit in more with how core does things.

```javascript
import { __ } from '@wordpress/i18n';
// ...

function() {
   // ...
   { __( 'Build your audience with tags' ) }
}
```

Translations in core also have this thing called "domain", and it's so it can distinguish between strings that are identical but from different plugins.

If we moved this to the editing tookit plugin we'd do `__( 'Build your audience with tags', 'full-site-editing' )` (because the plugin used to be called `full-site-editing` so that's the domain we use for translations)